sdk/eventhub/azure-eventhubs/azure/eventhub/receiver.py

```diff
@@ -264,7 +269,7 @@ def queue_size(self):
             return self._handler._received_messages.qsize()
         return 0
 
-    def receive(self, max_batch_size=None, timeout=None):
+    def receive(self, max_batch_size=None, timeout=None, max_reconnect_retries=None):
```

Comment:
This new parameter only seems to get added to the synchronous receiver and not the async one?

Reply:
Ah, thank you!  I was too focused on the sync EPH vertical.